nexus/src/db/model/console_session.rs

```diff
@@ -27,3 +27,10 @@ impl ConsoleSession {
         self.token.clone()
     }
 }
+
+/// A console session with the silo id of the authenticated user
+#[derive(Clone, Debug)]
+pub struct ConsoleSessionWithSiloId {
+    pub console_session: ConsoleSession,
+    pub silo_id: Uuid,
+}
```

Comment:
I think it's fine for this to be defined in Nexus - maybe even here - but I want to point out that it's a little quirky to have this struct contained within `db::model`, when it's not actually used as a database model.

Reply:
eh... I guess this current location is fine. It *is* being returned from the `datastore` functions, so it probably still makes sense to live in `db` somewhere, and `db::model::console_session` is probably equivalent to having it exist in `db::datastore::console_session` - that's the only other place I'd consider placing it, since that's where the struct is created.